gateway/ln-gateway/proto/gateway_lnrpc.proto

```diff
@@ -332,4 +332,6 @@ message GetBalancesResponse {
 
     // The total balance of the node's lightning wallet, in millisats.
     uint64 lightning_balance_msats = 2;
+    // the total inbound Liquidity balance
```

Comment:
Nit: add newline above here to be consistent with the rest of the file

Reply:
> gateway_lnrpc.proto:335: trailing whitespace

You need to remove the trailing whitespace before committing. Your editor probably added it when you added a new line.